portfolio/pom.xml

```diff
@@ -23,6 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
If you're manually creating the JSON, you probably don't need to add this dependency